app/src/main/res/values/strings.xml

```diff
@@ -229,18 +229,18 @@
     <string name="backup_app_stopped_button">Launch</string>
     <string name="backup_app_stopped_no_intent">Cannot launch app</string>
     <!-- This text gets shown for apps that the OS did not try to backup for whatever reason e.g. no backup was run yet -->
-    <string name="backup_app_not_yet_backed_up">Waiting to back up…</string>
+    <string name="backup_app_not_yet_backed_up">Awaiting backup…</string>
     <string name="restore_app_not_yet_backed_up">Was not yet backed up</string>
     <string name="backup_app_in_progress">Backing up</string>
     <string name="backup_app_success">Backed up</string>
     <string name="backup_app_warning">Backup warning</string>
-    <string name="backup_app_was_stopped">Not backed up as it wasn\'t used recently</string>
-    <string name="restore_app_was_stopped">Was not backed up as it hadn\'t been used recently</string>
+    <string name="backup_app_was_stopped">Not backed up as it is force-stopped</string>
+    <string name="restore_app_was_stopped">Was not backed up as it was force-stopped</string>
```

Comment:
It is not clear that this change is correct.   Say what is really happening.  Perhaps "Not backed up because app is not running."

Reply:
Isn't the definition of running in Android kind of all over the place? It's not really clear to the user what each thing means, it may not be actively running or not, and it can still be backed up, no?

I feel like oversimplifying might be hurtful here